portfolio/src/main/webapp/tasksApi.js

```diff
@@ -102,6 +54,11 @@ function importTasklist(tasklistId) {
  * logged in.
  */
 function drawImportMenu() {
+  importMenuVisible = true;
+
+  const $button = $('#connect-tasks-btn')[0];
```

Comment:
Just a small suggestion, the variable name button seems a little vague.

Reply:
Done.